### PR TITLE
Add video autoplay option, fix playback time discrepancy

### PR DESCRIPTION
**Pull request by mrinalini-m**
1. Video wasn't starting at 0 when the lightbox was closed and reopened. Fixed it to match behavior on clicking prev, next.
2. Added option to autoplay video on lightbox open.
3. Add event onClosed.

Relevant: Includes code from this pull req -->  https://github.com/pexea12/vue-image-lightbox/pull/86

### CODE REVIEW

**Comment by shoeseat** [comment]
I am in version '7.0.0' and need autoplay feature in video playback. When this feature will be merged in new version?

**Comment by pexea12** [comment]
I will check the pull request in a few hours

**Comment by shoeseat** [comment]
@pexea12 Can you review and merge it if it's all good?

**Comment by pexea12** [comment]
I actually commented some changes but the author of the merge request didn't respond @shoeseat

**Comment by janosrusiczki** [comment]
@pexea12 Where did you comment those changes? I don't see any review on this PR.

**Comment by pexea12** [comment]
Ah, I have to press `Submit review`, I thought the reviews are there for weeks. Sorry guys

File: src/components/LightBox.vue [hunk 1]
```diff
@@ -159,70 +163,69 @@ if (typeof window !== 'undefined') {
...
 }
 
-
 export default {
   components: {
     LeftArrowIcon,
     RightArrowIcon,
     CloseIcon,
-    VideoIcon,
+    VideoIcon
```
[the remark this answers is not in the document]
Comment (mrinalini-m) [reply src/components/LightBox.vue:171]: Okay Done. I'd recommend adding some lint rules so you can run `yarn lint --fix`.

File: src/components/LightBox.vue [hunk 2]
```diff
@@ -355,30 +358,41 @@ export default {
...
+      document.querySelector('body').classList.remove('vue-lb-open')
+      document.removeEventListener('keydown', this.addKeyEvent)
 
-      if (value) {
-        document.addEventListener('keydown', this.addKeyEvent)
-      } else {
-        document.removeEventListener('keydown', this.addKeyEvent)
+      if (this.$refs.video) {
+        this.$refs.video.pause()
+        this.$refs.video.currentTime = '0'
```
[the remark this answers is not in the document]
Comment (mrinalini-m) [reply src/components/LightBox.vue:387]: Done.

**Comment by pexea12** [comment]
https://github.com/pexea12/vue-image-lightbox/releases/tag/7.1.0

**Pull request merged**